client/src/hooks/useForgotPasswordForm.jsx

```diff
@@ -0,0 +1,80 @@
+import { useDisclosure, useToast } from '@chakra-ui/react';
+import { useRef, useState } from 'react';
+import useEmailValidation from 'hooks/useEmailValidation.jsx';
+import { userForgotPassword } from 'utils/api/users.js';
+
+const useForgotPasswordForm = () => {
+  const emailRef = useRef('');
+  const [emailIsValidated, setEmailIsValidated] = useState(true);
+  const [humanIsValidated, setHumanIsValidated] = useState(true);
+  const [buttonIsDisabled, setButtonIsDisabled] = useState(true);
+  const toast = useToast();
+  const [isLoading, setIsLoading] = useState(false);
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const { isAnEmail } = useEmailValidation();
+
+  const forgotPasswordHandleSubmit = async (e) => {
+    e.preventDefault();
+    setIsLoading(true);
+
+    const email = emailRef.current.value;
+    const url = window.location.href.substr(0, window.location.href.lastIndexOf('/'));
```

Comment:
```suggestion
    const url = window.location.origin;
```

You always want to get the base URL right?

Reply:
yes! thats exactly what I am looking for, its fixed